projects/packages/my-jetpack/src/products/class-starter.php

```diff
@@ -226,6 +172,15 @@ public static function get_supported_products() {
 		return array( 'backup', 'anti-spam' );
 	}
 
+	/**
+	 * Return all the products it contains.
```

Comment:
This annotation seems to be the same as the function `get_supported_products` above.